src/trackerhandle.py

```
@@ -179,17 +191,21 @@ async def process_single_tracker(tracker):
                     ptp = PTP(config=config)
                     groupID = meta.get('ptp_groupID', None)
                     ptpUrl, ptpData = await ptp.fill_upload_form(groupID, meta)
+                    is_uploaded = False
                     try:
                         upload_start_time = time.time()
-                        await ptp.upload(meta, ptpUrl, ptpData, disctype)
+                        is_uploaded = await ptp.upload(meta, ptpUrl, ptpData, disctype)
                         upload_duration = time.time() - upload_start_time
                         meta[f'{tracker}_upload_duration'] = upload_duration
                         await asyncio.sleep(5)
                     except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)